recipe/meta.yaml

```diff
@@ -37,8 +39,10 @@ about:
     This provides a plugin for Hatch that uses your preferred version
     control system (like Git) to determine project versions.
   dev_url: https://github.com/ofek/hatch-vcs
+  doc_url: https://pypi.org/project/hatch-vcs/
   license: MIT
-  license_file: LICENSE.txt
```

Comment:
The license file does seem to be `LICENSE.txt` as seen here:
https://github.com/ofek/hatch-vcs/blob/v0.2.0/LICENSE.txt

Reply:
Done! Thanks!